thrust/thrust/functional.h

```diff
@@ -1606,8 +1606,7 @@ _CCCL_SUPPRESS_DEPRECATED_PUSH
  *  \param pred The Adaptable Binary Predicate to negate.
  *  \return A new object, <tt>npred</tt> such that <tt>npred(x,y)</tt> always returns
  *          the same value as <tt>!pred(x,y)</tt>.
- *  \tparam Binary Predicate is a model of <a
- * href="https://en.cppreference.com/w/cpp/utility/functional/AdaptableBinaryPredicate">Adaptable Binary Predicate</a>.
+ *  \tparam Binary Predicate is a model of an Adaptable Binary Predicate.
```

Comment:
There are mention of `AdaptableBinaryPredicate` on some Boost docs sites but absolutely no current hits on `cppreference`

Reply:
We could point to this model https://en.cppreference.com/w/cpp/utility/functional/binary_function

Reply:
Please don't :D Or just do it. The entity this documentation refers to was deprecated by me, since its equivalent C++ facility is deprecated since C++11 and was removed in C++17. I want to remove it from Thrust as well (soon). Users should migrate away from `thrust::binary_function`, which is also deprecated already.